docs/source/user_guide/automl.ipynb

```diff
@@ -318,6 +318,35 @@
    "source": [
     "automl.results"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ensemble methods: Stacking\n",
+    "\n",
+    "[Stacking](https://en.wikipedia.org/wiki/Ensemble_learning#Stacking) is an ensemble machine learning algorithm that involves training a model to best combine the predictions of several base learning algorithms. First, each base learning algorithms is trained using the given data. The combining algorithm or meta-learner is then trained on the predictions made by those base learning algorithms to make a final prediction.\n",
+    "\n",
+    "AutoML enables stacking using the `ensembling` flag during initalization; this is set to `False` by default. The stacking ensemble pipeline runs in its own batch after a whole cycle of training has occurred (each allowed pipeline trains for one batch). Note that this means a large number of iterations may need to run before the stacking ensemble runs. It is also important to note that only the first CV fold is calculated for stacking ensembles because the model internally uses CV folds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "X, y = evalml.demos.load_breast_cancer()\n",
+    "X_dt = ww.DataTable(X)\n",
+    "y_dc = ww.DataColumn(y)\n",
+    "automl_with_ensembling = AutoMLSearch(problem_type=\"binary\",\n",
+    "                                      allowed_model_families=[ModelFamily.RANDOM_FOREST, ModelFamily.LINEAR_MODEL],\n",
+    "                                      max_iterations=20,\n",
+    "                                      ensembling=True)\n",
+    "automl_with_ensembling.search(X_dt, y_dc)"
```

Comment:
Is it possible to add a call to the `stacked_ensembler.describe` so that users can see and better understand how this works? I think it would be helpful to include info on the other models that help make up this model's predictions